mm2src/mm2_main/src/database/my_swaps.rs

```diff
@@ -57,7 +57,7 @@ fn insert_saved_swap_sql(swap: SavedSwap) -> Option<(&'static str, Vec<String>)>
 #[derive(Debug)]
 pub enum SelectRecentSwapsUuidsErr {
     Sql(SqlError),
-    Parse(uuid::parser::ParseError),
+    Parse(uuid::Error),
```

Comment:
`uuid::Error` is used in multiple places already so it's better to import once

```rust
use uuid::Error as UuidError
```

Reply:
Done